pkg/controller/servicebindingrequest/reconciler.go

```diff
@@ -329,6 +355,70 @@ func (r *Reconciler) Reconcile(request reconcile.Request) (reconcile.Result, err
 	// splitting instance from it's status
 	sbrStatus := &sbr.Status
 
+	if sbrStatus.Conditions == nil {
```

Comment:
It is fine to omit this block; missing conditions are semantically equivalent to Unknown conditions.